Description: Do not ship compatibility symlinks.
 The old names were never used in an official Debian package,
 no no need to introduce them in Debian.
Author: Petter Reinholdtsen <pere@debian.org>
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2017-08-08

--- gr-gsm-0.0.20170807.6c4304d.orig/apps/CMakeLists.txt
+++ gr-gsm-0.0.20170807.6c4304d/apps/CMakeLists.txt
@@ -18,7 +18,6 @@
 # Boston, MA 02110-1301, USA.
 
 include(GrPython)
-include(CreateSymlink)
 add_subdirectory(helpers)
 
 GR_PYTHON_INSTALL(
@@ -29,15 +28,8 @@ GR_PYTHON_INSTALL(
     DESTINATION bin
 )
 
-CREATE_SYMLINK(grgsm_livemon airprobe_rtlsdr.py)
-CREATE_SYMLINK(grgsm_decode airprobe_decode.py)
-CREATE_SYMLINK(grgsm_scanner airprobe_rtlsdr_scanner.py)
-
-add_custom_target(apps_symlinks ALL DEPENDS ${symlinks})
-
 install(
     PROGRAMS
-    ${symlinks}
     DESTINATION bin
 )
 
--- gr-gsm-0.0.20170807.6c4304d.orig/apps/helpers/CMakeLists.txt
+++ gr-gsm-0.0.20170807.6c4304d/apps/helpers/CMakeLists.txt
@@ -27,13 +27,8 @@ GR_PYTHON_INSTALL(
     DESTINATION bin
 )
 
-CREATE_SYMLINK(grgsm_capture.py airprobe_rtlsdr_capture.py)
-
-add_custom_target(helpers_symlinks ALL DEPENDS ${symlinks})
-
 install(
     PROGRAMS
-    ${symlinks}
     DESTINATION bin
 )
 
